core/arch/aarch64/proc.c

```diff
@@ -218,10 +226,17 @@ proc_init_arch(void)
         LOG_FEATURE(FEATURE_MTE);
         LOG_FEATURE(FEATURE_MTE2);
         LOG_FEATURE(FEATURE_BTI);
+        LOG_FEATURE(FEATURE_SSBS);
+        LOG_FEATURE(FEATURE_SSBS2);
 
         LOG(GLOBAL, LOG_TOP, 1, "Processor features:\n ID_AA64ISAR2_EL1 = 0x%016lx\n",
             cpu_info.features.isa_features[AA64ISAR2]);
         LOG_FEATURE(FEATURE_PAUTH2);
+        LOG_FEATURE(FEATURE_CONSTPACFIELD);
+
+        LOG(GLOBAL, LOG_TOP, 1, "Processor features:\n ID_AA64MMFR2_EL1 = 0x%016lx\n",
```

Comment:
Do we need the \n after "features:"?  Makes it harder to search the log with an entry split across lines.  Ditto above.  Was this supposed to a single "Processor features:\n" instead of prefixing every individual feature register?

Reply:
`Processor features:` uses `\n` in order to group features by feature register, using indented `Processor has` without `\n` for each feature in the register, e.g.
```
Running on a 16 CPU machine
Processor features:
 ID_AA64ISAR0_EL1 = 0x1011111110212120
   Processor has FEATURE_AESX
   Processor has FEATURE_PMULL
   Processor has FEATURE_SHA1
   Processor has FEATURE_SHA256
   Processor has FEATURE_SHA512
   Processor has FEATURE_CRC32
   Processor has FEATURE_LSE
   Processor has FEATURE_RDM
   Processor has FEATURE_SHA3
   Processor has FEATURE_SM3
   Processor has FEATURE_SM4
   Processor has FEATURE_DotProd
   Processor has FEATURE_FHM
   Processor has FEATURE_FlagM
   Processor has FEATURE_RNG
Processor features:
 ID_AA64ISAR1_EL1 = 0x0011100001211032
   Processor has FEATURE_DPB
   Processor has FEATURE_DPB2
Processor features:
 ID_AA64PFR0_EL1 = 0x0001000100110011
   Processor has FEATURE_FP16
   Processor has FEATURE_SVE
Processor features:
 ID_AA64MMFR1_EL1 = 0x0000000000000000
Processor features:
 ID_AA64DFR0_EL1 = 0x0000000000000006
   Processor has FEATURE_LRCPC
Cache line size is 64 bytes
```

Reply:
Hmm, it still seems like only the first "Processor features:" is needed?